test/e2e/runner/perturb.go

```diff
@@ -48,11 +44,11 @@ func PerturbNode(node *e2e.Node, perturbation e2e.Perturbation) (*rpctypes.Resul
 	switch perturbation {
 	case e2e.PerturbationDisconnect:
 		logger.Info("perturb node", "msg", log.NewLazySprintf("Disconnecting node %v...", node.Name))
-		if err := execDocker("network", "disconnect", testnetName, name); err != nil {
+		if err := execDocker("network", "disconnect", testnet.Name+"_"+testnet.Name, name); err != nil {
```

Comment:
[no change needed] why use the testnet name twice in this string? Before it used the `baseDir_testnet.Name` but now it seems repetitive for no reason. Perhaps this works?

```suggestion
		if err := execDocker("network", "disconnect", testnet.Name, name); err != nil {
```